ts/Dashboards/Utilities.ts

```diff
@@ -221,6 +221,39 @@ function merge<T extends Object>(
     return obj;
 }
 
+/**
+ * Returns a deep copy of an argument. It differs from `merge` in that it copies
+ * also arrays.
+ *
+ * @param value
+ * The value to clone.
+ *
+ * @param excludedKeys
+ * An array of keys to exclude from the clone.
+ */
+function deepClone(value: any, excludedKeys?: string[]): any {
```

Comment:
`Object.assign({},..)` will not be easier?

Reply:
No, because it does only a shallow copy, not a deep one.